lib/nfg_ui/components/patterns/modal.rb

```diff
@@ -10,7 +10,14 @@ class Modal < NfgUi::Bootstrap::Components::Modal
         include NfgUi::Components::Utilities::Renderable
         include NfgUi::Components::Utilities::Titleable
 
+        include NfgUi::Components::Traits::Modal
         include NfgUi::Components::Traits::Size
+
+        private
+
+        def resized?
+          [:sm, :lg, :xl, :cover].include?(size)
+        end
```

Comment:
@farhajawed modal component can accept `:cover` trait as indicated here.

Reply:
Here: https://github.com/network-for-good/nfg_ui_display_app/pull/1/files#r669083130